pidcat.py

```diff
@@ -310,4 +374,4 @@ def parse_start_proc(line):
     message = matcher.sub(replace, message)
 
   linebuf += indent_wrap(message)
-  print(linebuf.encode('utf-8'))
+  print linebuf.encode('utf-8'), "\r"
```

Comment:
I tried the following, but it did not work. It would not print out the colors correctly.
linebuf += indent_wrap(message)
linebuf += "\n"
print(linebuf.encode('utf-8'))

I couldn't get any variation to work. Not sure why not.
I could only make it work by removing the utf-8.

Suggestions?